action.yml

```diff
@@ -0,0 +1,43 @@
+name: Uno.Check
+description: "GitHub Action to setup a remote environment to build Uno Platform apps"
+
+branding:
+  icon: 'download'
+  color: 'red'
+
+inputs:
+  target-platform:
+    description: 'The platform to install dependencies for. See available values at https://platform.uno/docs/articles/external/uno.check/doc/using-uno-check.html'
+    required: false
+    default: 'all'
+  dotnet-version:
+    description: 'Installs and sets the .NET SDK Version'
+    required: false
+    default: '9.0.x'
+  sdkVersion:
+    description: 'The version of the Windows Sdk'
+    required: false
+    default: '19041'
```

Comment:
```suggestion
```

Reply:
Good catch, I forgot to remove that
Thanks!